paket.dependencies

```diff
@@ -16,13 +20,14 @@ nuget Ionide.KeepAChangelog.Tasks copy_local:true
 nuget Expecto
 nuget Expecto.Diff
 nuget Expecto.TestResults
-nuget MedallionShell ~> 1.5.0
+nuget MedallionShell
 
 # copy_local: false excludes runtime assets (desired for msbuild)
-nuget Microsoft.Build copy_local: false
-nuget Microsoft.Build.Framework copy_local: false
-nuget Microsoft.Build.Utilities.Core copy_local: false
-nuget Microsoft.Build.Tasks.Core copy_local: false
+nuget Microsoft.Build  17.4.0-preview-22469-04 copy_local: false
+nuget Microsoft.Build.Framework 17.4.0-preview-22469-04  copy_local: false
+nuget Microsoft.Build.Utilities.Core 17.4.0-preview-22469-04  copy_local: false
+nuget Microsoft.Build.Tasks.Core 17.4.0-preview-22469-04  copy_local: false
+nuget Microsoft.NET.StringTools 17.4.0-preview-22469-04
```

Comment:
Had to specify all these exactly because paket wasn't downloading them for some reason.